pyproject.toml

```diff
@@ -27,40 +34,156 @@ classifiers = [
     "Topic :: Scientific/Engineering :: Physics",
     "Topic :: Software Development :: Libraries :: Python Modules"
 ]
+dependencies = [
+    "pydantic==2.5.*",
+    "h5py==3.11.*",
+    "numpy==1.26.*",
+    "Pillow==10.3.*",
+]
+dynamic = [
+    "version"
+]
+
+[project.optional-dependencies]
+test = [
+    "coverage",
+    "pytest",
+    "pytest-cov",
+    "scipy==1.14.*",
+]
+quality = [
+    "pylint",
+    "pre-commit",
+]
+docs = [
+    # Sphinx dependencies
+    "sphinx",
+    "sphinx_issues",
+    "sphinx-autodoc-typehints",
+    "sphinx-rtd-theme",
```

Comment:
I'm pretty sure, it's not used because in `readthedocs` used the following configuration:

```
    - sphinx
    - sphinx_issues
    - sphinx_rtd_theme
    - sphinxcontrib-bibtex
    - sphinxcontrib-googleanalytics
    - nbsphinx
    - nbstripout
    - numpydoc
    - jupyter_client
```